src/interfaces/interfaces.tsx

```diff
@@ -0,0 +1,34 @@
+
+export interface UserInterface {
+  id: number;
+  firstName: string;
+  lastName: string;
+  username: string;
+  orders: ItemInterface[];
+  createdDate: string;
+}
+
+export interface ItemInterface {
+  id: number;
+  title: string;
+  description: string;
+  category: string;
+  photos: string[];
+  rating: number;
+  sold: boolean;
+  userId: number;
+  createdDate: string;
+  price: number;
+}
+
+export interface CartInterface {
```

Comment:
de regula nu se foloseste cuvantul `Interface` in denumiriile de interface. Sugerez sa modifici la `Entity` sau sa scoti complet